packages/web/src/components/data-entry/DropdownInput.module.css

```diff
@@ -230,3 +233,11 @@
   line-height: 28px;
   height: 32px;
 }
+
+.select .footer:global(.ant-select-item-option) {
+  cursor: default;
+  margin: 0 -8px;
+  padding: 0px !important;
```

Comment:
smell here, but antd styling is weird. don't want to spend too much time on this at this particular time